docs/src/content/budget.mdx

```diff
@@ -0,0 +1,136 @@
+
+* At least 100 active Icebreakers by end of year, 2025
+* 100% of transactions submitted through Blockfrost will be handled by the Icebreakers by end of year, 2025
+* Quarterly updates provided on the SPO call
+* Monthly community development call
+
+
+## Roadmap & Timelines
+
+The Blockfrost [Roadmap](https://github.com/orgs/blockfrost/projects/17/views/1) ([roadmap view](https://github.com/orgs/blockfrost/projects/17/views/2)) is publically available on our Github and has been structured into distinct eras.
+
+You can also watch the [walkthrough video on our roadmap](https://www.youtube.com/watch?v=5RedwIYmAgc).
+
+All milestones are clearly documented and tracked on our Github roadmap to promote accountability and transparancy. 
+
+#### Icebreakers era
+
+* [Icebreakers: Blockfrost platform initial release](https://github.com/blockfrost/blockfrost-platform/issues/305)
+* [Icebreakers: Transaction submission endpoint](https://github.com/blockfrost/blockfrost-platform/issues/274)
+* [Icebreakers: Platform gateway](https://github.com/blockfrost/blockfrost-platform/issues/276)
+* [Icebreakers: Onboarding](https://github.com/blockfrost/blockfrost-platform/issues/284)
+* [Icebreakers: Live ledger endpoints](https://github.com/blockfrost/blockfrost-platform/issues/275)
+* [Icebreakers: Infrastructure deployment](https://github.com/blockfrost/blockfrost-platform/issues/281)
+* [Icebreakers: Data nodes support](https://github.com/blockfrost/blockfrost-platform/issues/279)
+* [Icebreakers: Conformance testing](https://github.com/blockfrost/blockfrost-platform/issues/304)
+* [Icebreakers: Caching and invalidation](https://github.com/blockfrost/blockfrost-platform/issues/280)
+
+#### Hopper era
+
+* [Hopper: Research into payments channels between the platform and the consumer](https://github.com/blockfrost/blockfrost-platform/issues/287)
+* [Hopper: Hydra payment channels between platform and gateway](https://github.com/blockfrost/blockfrost-platform/issues/277)
+* [Hopper: Hydra payment channel between gateway and consumer](https://github.com/blockfrost/blockfrost-platform/issues/278)
+
+
+## Position with the Cardano Vision and Roadmap for 2025
+
+In March, the majority of DReps voted to adopt the strategic document [Cardano Vision and Roadmap for 2025](https://adastat.net/governances/56f39054758f1a3cedc1de9225d66bf270b62dfdbfbc5399f1d6d43aceffc63600).
+
+Our proposal directly addresses the document's item:
+
+  "Decentralize data API services to reduce reliance on centralized providers and empower SPOs."
+  
+We are decentralizing the Blockfrost API and increasing the SPO incentives, at the same time.
+
+## Proposed budget
+
+The majority of our budget is allocated to developer salaries. We are requesting an annual budget of $120,000  to support one full-time employee (FTE).
+
+Our total funding request from the treasury is $650,000, which is equivalent to approximately 988,708 ADA based on the current rate of 0.65 USD per ADA.
```

Comment:
```suggestion
Our total funding request from the treasury is $650,000, which is equivalent to approximately 1,300,000 ADA based on the current rate of 0.50 USD per ADA.
```

Looks like most of the proposals are counting with half a dollar per ADA as a conservative estimate.